### PR TITLE
remove default GTNH tank recipes

### PR DESCRIPTION
**Pull request by bombcar**
we override these anyway elsewhere

also add NEI to runClient so you can do things

### CODE REVIEW

File: src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java [hunk 1]
```diff
@@ -60,74 +55,26 @@ public void initFirst() {
...
-    		if (((EnumMachineEta) type).register()) {
-                addColorRecipes(type);
-                CraftingPlugin.addShapedRecipe(getColorTank(type, EnumColor.WHITE, 8), recipe);
-                return true;
-            }
-    	}       
-        return false;
-    }
-    
-    private void addColorRecipes(IEnumMachine type) {
```
Comment (Dream-Master) [inline src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java:108]: The color recipes i would keep

**Comment by Dream-Master** [comment]
remove recipes only is ok so i can remove the script recipe and color tank recipes are visible for iron and steel

**Comment by bombcar** [comment]
ok will add the colors back in

**Comment by bombcar** [comment]
wait these were color recipes for everything BUT iron and steel I think ... checking

**Comment by draknyte1** [comment]
> wait these were color recipes for everything BUT iron and steel I think ... checking

correct.

**Comment by Dream-Master** [comment]
if you remove the recipe in rc code i can remove the script recipe (tank recipe remove) and color recipes back there

File: src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java [hunk 2]
```diff
@@ -33,8 +28,8 @@
     public final static int CAPACITY_PER_BLOCK_OSMIUM = 2048 * FluidHelper.BUCKET_VOLUME;
     public final static int CAPACITY_PER_BLOCK_NEUTRONIUM = 3072 * FluidHelper.BUCKET_VOLUME;
 
-    public static final HashMap<String, IEnumMachine> cacheTankType = new HashMap<String, IEnumMachine>();
-    public static final HashMap<String, GenericMultiTankBase> cacheTankMaterial = new HashMap<String, GenericMultiTankBase>();
+    public static final HashMap<String, IEnumMachine> cacheTankType = new HashMap<>();
```
Comment (draknyte1) [inline src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java:31]: Why remove the map types?

Comment (bombcar) [reply src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java:31]: It's implicit from the HashMap but I can put them back in

Comment (draknyte1) [reply src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java:31]: >Implicit

Not really.
Your IDE will/should complain the typing is missing.
image

Comment (bombcar) [reply src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java]: my IDE complains it's explicit hehehe 
image
reverted tho

File: src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java (hunk 1 above)
Comment (bombcar) [reply src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java]: ok put them back in

**Comment by bombcar** [comment]
this revert broke things, hold

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**